ide/plugins/java-format/src/content/formats/webdriver-testng.js

```diff
@@ -0,0 +1,523 @@
+/*
+ * Formatter for Selenium 2 / WebDriver Java client.
+ */
+
+if (!this.formatterType) {  // this.formatterType is defined for the new Formatter system
+  // This method (the if block) of loading the formatter type is deprecated.
+  // For new formatters, simply specify the type in the addPluginProvidedFormatter() and omit this
+  // if block in your formatter.
+  var subScriptLoader = Components.classes["@mozilla.org/moz/jssubscript-loader;1"].getService(Components.interfaces.mozIJSSubScriptLoader);
+  subScriptLoader.loadSubScript('chrome://selenium-ide/content/formats/webdriver.js', this);
+}
+
+function useSeparateEqualsForArray() {
+  return true;
+}
+
+function testClassName(testName) {
+  return testName.split(/[^0-9A-Za-z]+/).map(
+      function(x) {
+        return capitalize(x);
+      }).join('');
+}
+
+function testMethodName(testName) {
+  return "test" + testClassName(testName);
+}
+
+function nonBreakingSpace() {
+  return "\"\\u00a0\"";
+}
+
+function array(value) {
+  var str = 'new String[] {';
+  for (var i = 0; i < value.length; i++) {
+    str += string(value[i]);
+    if (i < value.length - 1) str += ", ";
```

Comment:
Fixed in [#7e0de8f](https://github.com/rasika90/selenium/commit/7e0de8f7253c45ace8d0fd539f4712e1f91349c1).